src/NuGet.Clients/NuGet.PackageManagement.PowerShellCmdlets/Cmdlets/GetProjectCommand.cs

```diff
@@ -50,24 +52,26 @@ protected override void ProcessRecordCore()
 
             if (All.IsPresent)
             {
-                var projects = EnvDTESolutionUtility.GetAllEnvDTEProjects(DTE);
+                VsSolutionManager.EnsureSolutionIsLoaded();
+                var projects = VsSolutionManager.GetAllVsProjectAdapters().Select(p => p.Project);
```

Comment:
With `-all` switch it retrieves all of DTE projects. SO it's more efficient to force entire solution to load before rather than separate projects one-by-one.